jekyll/_ccie/troubleshooting-install.md

```diff
@@ -13,6 +13,62 @@ published: true
+
+in `/etc/circle-installation-customizations` for it to work. You also won't need to have docker specified in the circle.yml as the container will have access to the docker socket in the background.
+
+#### Can I use a custom image in single box mode?
+
+Yes! You can do this by adding this to a file called `/etc/circle-installation-customizations`:
+
+`export CIRCLE_CONTAINER_IMAGE_URI="docker://circleci/build-image:latest"`
+That'll give you the same image that is currently used on circleci.com, but you can replace the URI image with anything that is currently pulled unto the machine or exists in dockerhub. This works with [custom images](https://github.com/circleci/image-builder) as well. 
+
+#### "Why isn't CircleCI using HTTPS?"
+
+While we create a self-signed cert when starting up, that certificate only applies to the management console and not the CircleCI product itself. If you want to use HTTPS, you'll have to give us certificates to use under the `Privacy` section of the settings in the management console.
+
+#### "Why doesn't terraform destroy every resource"
+
+We set the services box to have termination protection in AWS. We also write to an s3 bucket. If you want terraform to destroy every resource, you'll have to either manually delete the instance, or turn off termination protection in the circleci.tf file. You'll also need to empty the s3 bucket that was created as part of the terraform install.'
+
+#### "How do I backup CircleCI?"
+
+As of 1.48.0 we've enabled you to take snapshots right in the management console, and follow the steps below:
+
+1: On the management console, https://services_ip:8800, create a snapshot
+
+2: After it completes, backup the full snapshots directory `/var/lib/replicated/snapshots`, including the json file, to something safe.
+
+3: Take down the services box and rebuild it, but do not run the setup wizard yet.
+
+4: Copy the snapshots directory to the services box. I recomend `/home/ubuntu/ so to avoid permissions issues.
+
+5: Run the startup. When it asks for the licence, instead click `restore from a snapshot` 
+
+6: Enter the path ie: `/home/ubuntu/snapshots` and click browse.
+
+7: Select the snapshot and choose restore.
+
+8: Restore the full snapshot and that will return your licence, console passphrase and databases.
+
+#### "How do I upgrade builder instances"
+
+The builder instances take their data directly from the services box when they initilize. Thus in order to update the builder boxes you'll need update the services box first, then create a new builder fleet ( or roll the builder fleet using an autoscaling group). During this start-up phase the builders will take the new update from the services machines, and things will be up-to-date.
+
+
+#### "Do the builders store any state?"
+
+The builders are stateless and only store docker images if they are set up that way. Otherwise, they can be torn down without worry as they don't persist any data.'
```

Comment:
Any docker images living on builders shouldn't be relied upon... I'd say you could just leave that out, or maybe clarify what "store docker images" means. They're effectively just a cache if present.